config/deploy.rb

```diff
@@ -0,0 +1,55 @@
+# config valid only for current version of Capistrano
+# lock '3.4.0'
+
+set :application, 'figgy'
+set :repo_url, 'https://github.com/pulibrary/figgy.git'
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+set :branch, ENV['BRANCH'] || 'master'
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, '/var/www/my_app_name'
+set :deploy_to, '/opt/figgy'
```

Comment:
Do we want to use `/opt/rails_app`?  I have mixed feelings about this: it's nice to be able to tell which app you're dealing with, but it's also nice to know the path regardless of which app it is.